site/.vitepress/sidebar/packages/aa-core.ts

```diff
@@ -143,102 +143,7 @@ export const aaCoreSidebar: DefaultTheme.SidebarItem = {
     {
       text: "Accounts",
       base: "/packages/aa-core/accounts",
-      collapsed: true,
-      items: [
-        {
-          text: "Introduction",
-          link: "/introduction",
-        },
-        {
-          text: "constructor",
-          link: "/constructor",
-        },
-        {
-          text: "Required Methods",
-          collapsed: true,
-          base: "/packages/aa-core/accounts/required",
-          items: [
-            {
-              text: "getDummySignature",
-              link: "/getDummySignature",
```

Comment:
@moldy530 to confirm, these are not removed, but moved to the sub site config of aa-core/accounts right?

Reply:
A number of them have been removed yea because they're covered in the root `toSmartAccount` docs and the rest yea were moved